pymoto/modules/assembly.py

```diff
@@ -245,28 +281,72 @@ class AssembleMass(AssembleGeneral):
 
     def _prepare(self, domain: DomainDefinition, *args, rho: float = 1.0, bcdiagval=0.0, **kwargs):
         # Element mass matrix
-        # 1/36 Mass of one element
-        mel = rho * np.prod(domain.element_size)
+        self.ME = ConsistentMassEq(domain, ndof=domain.dim, MP=rho)
```

Comment:
Idem; not clear what ME stands for, better use a descriptive name

Reply:
Agree; KE is also not descriptive. Imo it should be changed to "mass_element" and "stiffness_element" respectively.

Reply:
fixed for both in new commits